### PR TITLE
i18n(es-ES): translated `astro-syntax`

### PR DESCRIPTION
**Pull request by juanburg98**
[x] `astro-syntax` translated into Spanish

### CODE REVIEW

File: src/content/docs/es/core-concepts/astro-syntax.mdx [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+En Astro, se utiliza el formato estándar `kebab-case` para todos los atributos HTML en lugar del `camelCase` utilizado en JSX. Esto funciona incluso para `class`, que no está soportado por React.
+
+```jsx del={1} ins={2} title="ejemplo.astro"
+<div className="box" dataValue="3" />
+<div class="box" data-value="3" />
+```
+
+#### Comentarios
+
+En Astro, puede utilizar comentarios HTML estándar o comentarios de estilo JavaScript.
```
Comment (Waxer59) [inline src/content/docs/es/core-concepts/astro-syntax.mdx:178]:
```suggestion
En Astro, puedes utilizar comentarios HTML estándar o comentarios de estilo JavaScript.
```

File: src/content/docs/es/core-concepts/astro-syntax.mdx [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+```astro title="ejemplo.astro"
+---
+---
+<!-- La sintaxis de los comentarios HTML es válida en los archivos .astro -->
+{/* La sintaxis de los comentarios JS también es válida */}
+```
+
+:::caution
+Los comentarios HTML se incluirán en el DOM del navegador, mientras que los JS se omitirán. Para dejar mensajes TODO u otras explicaciones exclusivas de desarrollo, es posible que desee utilizar comentarios de estilo JavaScript en su lugar.
```
Comment (Waxer59) [inline src/content/docs/es/core-concepts/astro-syntax.mdx:188]:
```suggestion
Los comentarios HTML se incluirán en el DOM del navegador, mientras que los JS se omitirán. Para dejar mensajes TODO u otras explicaciones exclusivas de desarrollo, es posible que desees utilizar comentarios de estilo JavaScript en su lugar.
```

File: src/content/docs/es/core-concepts/astro-syntax.mdx [hunk 3]
```diff
@@ -0,0 +1,191 @@
+---
+title: Sintaxis de Astro
+description: Introducción a la sintaxis de los componentes .astro.
+i18nReady: true
+---
+
+**Si sabes HTML, ya sabes lo suficiente para escribir tu primer componente Astro.**
```
Comment (Waxer59) [inline src/content/docs/es/core-concepts/astro-syntax.mdx:7]:
```suggestion
**Si sabes HTML, ya sabes lo suficiente para escribir tu primer componente de Astro.**
```

File: src/content/docs/es/core-concepts/astro-syntax.mdx [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+
+La sintaxis de componentes Astro es un superconjunto de HTML. La sintaxis fue [diseñada para que resulte familiar a cualquiera que tenga experiencia escribiendo HTML o JSX](#diferencias-entre-astro-y-jsx), y añade soporte para incluir componentes y expresiones JavaScript.
+
+
+## Expresiones similares a JSX
+
+Puedes definir variables JavaScript locales dentro del script del componente frontmatter entre las dos vallas de código (`---`) de un componente Astro. A continuación, puede inyectar estas variables en la plantilla HTML del componente utilizando expresiones similares a JSX.
+
+:::note[dinámico frente a reactivo]
```
Comment (Waxer59) [inline src/content/docs/es/core-concepts/astro-syntax.mdx:16]:
```suggestion
:::note[dinámico vs reactivo]
```

File: src/content/docs/es/core-concepts/astro-syntax.mdx [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
+
+Al utilizar etiquetas dinámicas:
+
+- **Los nombres de las variables deben ir en mayúsculas.** Por ejemplo, usa `Elemento`, no `elemento`. De lo contrario, Astro intentará representar el nombre de la variable como una etiqueta HTML literal.
+
+- **Cuando se usan [`client:*` directivas de hidratación](/es/core-concepts/framework-components/#hidratando-componentes-interactivos), Astro necesita saber qué componentes empaquetar para producción, y el patrón de etiquetas dinámicas impide que esto funcione.
+
+### Fragmentos y elementos múltiples
+
+Una plantilla de componente Astro puede renderizar múltiples elementos sin necesidad de envolver todo en un único `<div>` o `<>`, a diferencia de JavaScript o JSX.
```
Comment (Waxer59) [inline src/content/docs/es/core-concepts/astro-syntax.mdx:127]:
```suggestion
Una plantilla de un componente de Astro puede renderizar múltiples elementos sin necesidad de envolver todo en un único `<div>` o `<>`, a diferencia de JavaScript o JSX.
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias.

> Aparte de que tendrias que arreglar los links rotos

**Comment by Yan-Thomas** [comment]
Thanks @juanburg98! This PR looks fine to me, but there are a few broken links, could you please take the time to solve them so we can merge it? 💜

**Comment by juanburg98** [comment]
@Yan-Thomas Everything is ready

**Comment by kevinzunigacuellar** [comment]
thanks everyone! @juanburg98, @Waxer59 and @Yan-Thomas 💜

**Pull request merged**